Robus/src/robus.c

```diff
@@ -33,7 +33,7 @@ typedef struct __attribute__((__packed__))
     };
 } node_bootstrap_t;
 
-#define NETWORK_TIMEOUT 100 // timeout to detect a failed detection
+#define NETWORK_TIMEOUT 1000 // timeout to detect a failed detection
```

Comment:
I think the function that use this define Robus_RunNetworkTimeout(void) is not needed anymore test if you can remove it